smithy-aws-protocol-tests/model/restXml/services/s3.smithy

```diff
@@ -380,6 +380,42 @@ structure GetBucketLocationOutput {
     LocationConstraint: BucketLocationConstraint,
 }
 
+@httpResponseTests([{
+            id: "GetObjectInvalidExpires",
+            documentation: """
+                S3 clients should not fail the request with invalid expires.
+                GA SDKs should verify the value in ExpiresString param and
+                new SDKs should verifiy the value in Expires param
+            """,
+            code: 200,
+            headers: {
+                "Expires": "foobar1234"
+            },
```

Comment:
Just an idea, but we could add an `ExpiresString` member to `GetObjectOutput`, and update this test to have the following:
```suggestion
            },
            params: {
                ExpiresString: "foobar1234"
            },
```